### ISSUE TITLE
How to bypass Windows kernel driver signature verification?

### ISSUE THREAD

**Issue by badboycxcc** [Tserith/Parasite#8]

**Comment by Tserith** [comment]
The bootkit bypasses it by hooking the signature checking code. This project is a PoC and will not work on all versions.

**Comment by ibay770** [comment]
Can this disable driver signature enforcement?

**Comment by Tserith** [comment]
Yes, it patches the bootloader's signature enforcement code to load an unsigned driver.
